src/components/NewsCarousel/NewsCarousel.module.scss

```diff
@@ -0,0 +1,43 @@
+@import 'styles/uswdsDependencies';
+@import 'styles/sfds/sfdsDependencies';
+
+.carouselContainer {
+  display: flex !important;
```

Comment:
might not need the `!important`? it renders correctly without it, but maybe there's an edge case i'm not seeing

Reply:
Got it! Yeah there is a good chance that I just forgot to remove this.